.rhcicd/clowdapp-backend.yaml

```diff
@@ -160,7 +160,7 @@ parameters:
   value: 500m
 - name: DB_CLEANER_SCHEDULE
   description: Execution time specified in cron format
-  value: "*/10 * * * *"
+  value: "0 1 * * *"
```

Comment:
The DB cleaning cron job will now start at 1:00 AM (one hour after the aggregation process) each day.